retromock/src/main/java/co/infinum/retromock/Retromock.java

```diff
@@ -240,8 +239,10 @@ public Object invoke(final Object proxy, final Method method, final Object[] arg
             return method.invoke(delegate, args);
           }
 
+          CallWrapper callWrapper = CallWrapperFactory.<T>create(method, args);
```

Comment:
In theory we could do that only once and save us time for parsing reflection parameters for each call of the method.
Check how RetromockMethod is created only once for each method and cached for later.
This will require to move `args` from constructor to wrap method.
Does it seems okay to you?

Reply:
Nice optimisation. Will do that 👍